applications/mlflow/chart/mlflow/templates/_helpers.tpl

```diff
@@ -0,0 +1,84 @@
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "mlflow.name" -}}
```

Comment:
Interesting why do these all have 'mlflow' as a prefix? Makes it hard to lift this into different charts.

Reply:
I think this is pretty common actually. I've typically done it this way so that you can distinguish between the potentially multiple micro-services in a single chart.